### PR TITLE
V22F-720 Edited hints + changed layout endscreen 

### PR DESCRIPTION
**Pull request by MaikVink**
(Eva's branch)
This branch checks the hints for the use of speed dreamy terms.
It also adds labels to the certificate buttons to make them more clear.
The generate button in the init of researcher menu is put in front of the apply for easy keyboard navigation

### CODE REVIEW

**Review by larsdekwant — CHANGES_REQUESTED** [review]
The following hints should be updated (in DatabaseSettingsMenu.xml)
- CACertDialog
- PublicCertDialog
- PrivateCertDialog

The hin for 'No, dont save' should be updated to start with a capital letter, same for the two hints on the menu after that.

File: speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DatabaseConnectionManager.cpp [hunk 1]
```diff
@@ -274,12 +274,16 @@ void SetSchema(void* p_scrHandle, int p_schemaControl)
 /// @param p_caControl the corresponding ui element control integers
 /// @param p_publicControl the corresponding ui element control integers
 /// @param p_privateControl the corresponding ui element control integers
-void SetUseSSL(tCheckBoxInfo* p_info, void* p_scrHandle, int p_caControl, int p_publicControl, int p_privateControl)
+void SetUseSSL(tCheckBoxInfo* p_info, void* p_scrHandle, int p_caControl, int p_publicControl, int p_privateControl,
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DatabaseConnectionManager.cpp:277]: missing new parameters in documentation

**Review by larsdekwant — CHANGES_REQUESTED** [review]
Missing documentation on newly added parameters and make sure to run clang-format on the 3 files suggested by the bot.

**Review by larsdekwant — APPROVED** [review]

**Review by suppergerrie2 — APPROVED** [review]

**Review by suppergerrie2 — CHANGES_REQUESTED** [review]
Add changes to Code changes document

**Pull request merged**